src/client/redux/reducers/cartItems.js

```diff
@@ -0,0 +1,14 @@
+import * as actionTypes from "../actionTypes";
+
+export default function cartItemsReducer(state = [], action) {
+  switch (action.type) {
```

Comment:
ADD_ITEMS_TO_CART_REQUEST and ADD_ITEMS_TO_CART_FAILURE are not handled in reducer.
I think,We should we also keep track of loading state in this reducer

Reply:
yes every reducer which is listening to an action related to an api call should listen to all the three actions. I did not add code here because I wanted to do that live in the workshop when I start adding use cases like delay in API response time or What i can do is make few more branches which have complete code. In workshop I will use this branch and attendees for final code can refer to complete branches.